catalogue/webapp/test/components/ItemRequestButton.test.js

```diff
@@ -0,0 +1,108 @@
+// @flow
+import {
+  mountWithTheme,
+  updateWrapperAsync,
+} from '@weco/common/test/fixtures/enzyme-helpers';
+import ItemRequestButton from '@weco/catalogue/components/ItemRequestButton/ItemRequestButton';
+import useAuth from '@weco/common/hooks/useAuth';
+import { getUserHolds } from '@weco/catalogue/services/stacks/requests';
+import mockAuthStates from '@weco/catalogue/__mocks__/auth-states';
+import mockWork from '@weco/catalogue/__mocks__/stacks-work';
+import mockRequests from '@weco/catalogue/__mocks__/stacks-requests';
+
+const availableItem = mockWork.items.find(
+  item => item.status.label === 'Available'
+);
+const unavailableItem = mockWork.items.find(
+  item => item.status.label !== 'Available'
+);
+const unrequestableItem = mockWork.items.find(
+  item => item.status.label === 'Missing'
+);
+
+jest.mock('@weco/catalogue/services/stacks/requests', () => ({
+  getUserHolds: jest.fn(),
+}));
+jest.mock('@weco/common/hooks/useAuth');
+
+describe('Feature: 2. As a library member I want to request an item', () => {
```

Comment:
Are these numbers important? It makes the code quite hard to read.

Reply:
They match the scenarios we wrote together here: https://github.com/wellcomecollection/wellcomecollection.org/blob/master/docs/scenarios/stacks.feature

We numbered them because it makes it easier to talk about them